services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/DirectionsResponseTest.java

```diff
@@ -41,15 +49,27 @@ public void fromJson_correctlyBuildsFromJson() throws Exception {
   }
 
   @Test
-  public void testToFromJson() throws Exception {
-    String json = loadJsonFixture(DIRECTIONS_V5_PRECISION6_FIXTURE);
-    DirectionsResponse responseFromJson1 = DirectionsResponse.fromJson(json);
+  public void testToFromJsonWithRealResponse() throws Exception {
+    Gson gson = new GsonBuilder().create();
+    String originalJson = loadJsonFixture(DIRECTIONS_V5_PRECISION6_FIXTURE_ARTIFICIAL_FIELDS);
```

Comment:
Added `DIRECTIONS_V5_PRECISION6_FIXTURE_ARTIFICIAL_FIELDS` just because `DirectionsResponse` always adds `routeIndex` and `requestUuid`.